src/components/MessagesList/MessagesGroup/MessagesSystemGroup.vue

```diff
@@ -119,8 +119,13 @@ export default {
 			immediate: true,
 			handler(value) {
 				this.messagesGroupedBySystemMessage = this.groupMessages(value)
+				this.updateCollapsedState()
```

Comment:
I'd go with something like this, to update reactive array only once
```suggestion
				this.messagesGroupedBySystemMessage = this.updateCollapsedState(this.groupMessages(value))
```